# The standard deviation

#### NOTE
The R progamming language uses the samples version of standard deviation and variance. The population version
is defined below.

In [1]:
pop.sd <- function(x){sd(x)*sqrt((length(x)-1)/length(x))}

## Problem-4-9
Find the standard deviation $s$ of each set of numbers in Problem 4.1.

### Solution-4-9

(a)
\begin{align}
\bar{X} & = \frac{\sum{X}}{N} = \frac{12+6+7+3+15+10+18+5}{8}=\frac{76}{8}=9.5 \\
\\
s       & = \sqrt{\frac{\sum{(X-\bar{X})^2}}{N}} \\
        & = \sqrt{\frac{(12-9.5)^2+(6-9.5)^2+(7-9.5)^2+(3-9.5)^2+(15-9.5)^2+(10-9.5)^2+(18-9.5)^2+(5-9.5)^2}{8}} \\
        & = \sqrt{23.75}=4.87
\end{align}

(b)
\begin{align}
\bar{X} & = \frac{9+3+8+8+9+8+9+18}{8}=\frac{72}{8}=9 \\
\\
s       & = \sqrt{\frac{\sum{(X-\bar{X})^2}}{N}} \\
        & = \sqrt{\frac{(9-9)^2+(3-9)^2+(8-9)^2+(8-9)^2+(9-9)^2+(8-9)^2+(9-9)^2+(18-9)^2}{8}} \\
        & = \sqrt{15}=3.87
\end{align}

In [4]:
p4.9 <- new.env()
p4.9$a.X <- c(12,6,7,3,15,10,18,5)
p4.9$b.X <- c(9,3,8,8,9,8,9,18)
p4.9$a.n <- length(p4.9$a.X)
p4.9$a.SD.pop <- round(pop.sd(p4.9$a.X),2)
p4.9$b.SD.pop <- round(pop.sd(p4.9$b.X),2)
cat("(a) Standard Deviation:", p4.9$a.SD.pop)
cat("\n(b) Standard Deviation:", p4.9$b.SD.pop)
sd(p4.9$a.X)
sd(p4.9$b.X)

(a) Standard Deviation: 4.87
(b) Standard Deviation: 3.87

[1] 5.209881

[1] 4.140393

The above results should be compared with those of Problem 4.3. It will be noted that the standard deviation does indicate that set (b) shows less dispersion than set (a). However, the effect is masked by the fact that extreme values affect the standard deviation much more than they affect the mean deviation. This is to be expected, of course, since the deviations are squared in computing the standard deviation.

---

## Problem-4-10
The standard deviation of two data sets given in Problem 4.1 were found using MINITAB and the results are shown below. Compare the answers with those obtained in Problem 4.9